javaparser-core/src/main/java/com/github/javaparser/ast/validator/language_level_validations/Java1_0Validator.java

```diff
@@ -138,4 +139,19 @@ public Java1_0Validator() {
         add(noSealedClasses);
         add(noPermitsListInClasses);
     }
+
+    /**
+     * Compose a validator message with suggestion to enable a feature.
+     * @param cause Original message - cause of the error.
+     * @param level Language level to enable.
+     * @return Composed message.
+     */
+    private static String message(final String cause, final ParserConfiguration.LanguageLevel level){
```

Comment:
Why a private static method?

Reply:
@jlerbsc Moved into a separate class `UpgradeJavaMessage` in order to use it in other Validators.